### PR TITLE
Refactor SslContextStore

### PR DESCRIPTION
**Pull request by mario-schwede-hivemq**
https://hivemq.kanbanize.com/ctrl_board/4/cards/8962/details/

### CODE REVIEW

File: src/test/java/com/hivemq/security/ssl/SslContextStoreTest.java [hunk 1]
```diff
@@ -140,18 +117,15 @@ public void test_put_at_start_throws_exception() throws Exception {
...
     }
 
     @Test
-    public void test_put_at_start() throws Exception {
+    public void test_create_at_start() throws Exception {
 
         final File keystore = keyStoreGenerator.generateKeyStore("test", "JKS", "pw", "pkpw");
 
-        final Tls tls = new Tls.Builder()
+        final Tls tls = new Tls.Builder() //
```
Comment (Remit) [inline src/test/java/com/hivemq/security/ssl/SslContextStoreTest.java:128]:
```suggestion
        final Tls tls = new Tls.Builder()
```

File: src/test/java/com/hivemq/security/ssl/SslContextStoreTest.java [hunk 2]
```diff
@@ -165,27 +139,24 @@ public void test_put_at_start() throws Exception {
...
+                .build());
     }
 
     @Test
     public void test_hash_key_and_trust_store() throws Exception {
         final File keystore = keyStoreGenerator.generateKeyStore("test", "JKS", "pw", "pkpw");
         final File trust = keyStoreGenerator.generateKeyStore("test", "JKS", "pw", "pkpw");
 
-        final Tls tls = new Tls.Builder()
+        final Tls tls = new Tls.Builder() //
```
Comment (Remit) [inline src/test/java/com/hivemq/security/ssl/SslContextStoreTest.java:159]:
```suggestion
        final Tls tls = new Tls.Builder()
```

File: src/test/java/com/hivemq/security/ssl/SslContextStoreTest.java [hunk 3]
```diff
@@ -200,30 +171,31 @@ public void test_hash_key_and_trust_store() throws Exception {
...
 
-        final FileOutputStream fileO = new FileOutputStream(trust, false);
-        final FileInputStream fileI = new FileInputStream(keyStoreGenerator.generateKeyStore("test", "JKS", "pw", "pkpw"));
-        final byte[] buf = new byte[1024];
-        while (fileI.read(buf) != -1) {
-            fileO.write(buf);
+        try (final FileInputStream input = new FileInputStream(keyStoreGenerator.generateKeyStore("test",
+                "JKS",
+                "pw",
+                "pkpw")); //
```
Comment (Remit) [inline src/test/java/com/hivemq/security/ssl/SslContextStoreTest.java:182]:
```suggestion
                "pkpw"));
```

File: src/test/java/util/TlsTestUtil.java [hunk 4]
```diff
@@ -20,32 +20,24 @@
...
 
-    public static @NotNull Tls createTLS(int timeout) {
+    public static @NotNull Tls.Builder createDefaultTLSBuilder() {
         return new Tls.Builder()
                 .withKeystorePath("")
                 .withKeystorePassword("")
                 .withKeystoreType("JKS")
                 .withPrivateKeyPassword("")
-                .withHandshakeTimeout(timeout)
+                .withHandshakeTimeout(10)
```
Comment (Remit) [inline src/test/java/util/TlsTestUtil.java:35]: It might make sense to provide the unit name in the name of the method

**Review by Remit — APPROVED** [review]

**Review by DC2-DanielKrueger — APPROVED** [review]

**Review by gitseti — APPROVED** [review]

**Pull request merged**